### PR TITLE
Add tooltips to project statuses

### PR DESCRIPTION
**Pull request by smcmo**
Noticed a lot of people asking about what the different statuses meant in the Discord, so I added tooltips that explain what they mean a bit more.

Example:
image

### CODE REVIEW

File: components/ui/Badge.vue [hunk 1]
```diff
@@ -14,20 +14,71 @@
...
+          'Your project is published, and can be accessed with a link, but cannot be seen in search or on your profile.'
+        "
+      >
+        <EyeOffIcon /> Unlisted
+      </span>
+    </template>
+    <template v-else-if="type === 'withheld'">
+      <span
+        v-tooltip="
+          'Your project has been withheld from publishing. See the moderation message for more details.'
```
Comment (triphora) [inline components/ui/Badge.vue:36]:
```suggestion
          'Your project is accessible by link but is not published in search. See the moderation message for more details.'
```

File: components/ui/Badge.vue [hunk 2]
```diff
@@ -14,20 +14,71 @@
...
+        v-tooltip="
+          'Your project cannot be found in search or your profile until you submit it for review.'
+        "
+        ><DraftIcon /> Draft
+      </span>
     </template>
-    <template v-else-if="type === 'draft'"><DraftIcon /> Draft</template>
     <template v-else-if="type === 'archived'">
-      <ArchiveIcon /> Archived
+      <span> <ArchiveIcon /> Archived </span>
```
Comment (triphora) [inline components/ui/Badge.vue:65]: What happened here?

**Review by triphora — CHANGES_REQUESTED** [review]

File: components/ui/Badge.vue (hunk 2 above)
Comment (smcmo) [reply components/ui/Badge.vue:65]: Oops missed that one

**Review by triphora — APPROVED** [review]

File: components/ui/Badge.vue [hunk 3]
```diff
@@ -14,20 +14,77 @@
...
+    <template v-else-if="type === 'private'">
+      <span v-tooltip="'Your project is only visible to members of your team.'">
+        <LockIcon /> Private
+      </span>
+    </template>
     <template v-else-if="type === 'scheduled'">
-      <CalendarIcon /> Scheduled
+      <span
+        v-tooltip="
+          'Your project will be automatically published at your chosen time and date.'
```
Comment (MagnusHJensen) [inline components/ui/Badge.vue:50]:
```suggestion
          'Your project will be automatically published at the chosen time and date.'
```

"Your" sounds more like it's an initial feature of scheduling, where as "the" sounds more like you have scheduled it.

**Comment by triphora** [comment]
Sorry for not getting to this sooner, but do you think you could redo this PR on [omorphia](https://github.com/modrinth/omorphia/blob/master/lib/components/base/Badge.vue)? I'm not sure if v-tooltip is working properly there yet, but this component will eventually be deleted in knossos once we convert it to using omorphia.

**Comment by smcmo** [comment]
I can definitely at least try it out yea

**Pull request closed without merging**